### PR TITLE
[JW8-11948] Bugfix/disable fullscreen pip

### PR DESCRIPTION
**Pull request by maniland**
Sets Fullscreen to false on pipLeave 

to fix the bug where the fullscreen icon stayed in an incorrect state when fullscreen was entered, then pip was enabled then disabled.

JW8-11948

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

**Comment by jwrobot-player** [comment]
[:warning:](https://jenkins-player.longtailvideo.com/job/jw7-pr-multi-opensource/1962/) MULTI Build for commit cf2eeafa65eb1c632082ceb13168248b58f22da2 is unstable (UNSTABLE).
[:building_construction:](https://jenkins-player.longtailvideo.com/job/jw7-opensource/9403/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-opensource/9403/consoleFull) jwplayer build SUCCESS
[:clipboard:](https://jenkins-player.longtailvideo.com/job/jw7-opensource-unit-tests/6329/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-opensource-unit-tests/6329/consoleFull) jwplayer unit tests SUCCESS
[:building_construction:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/28583/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/28583/consoleFull) jwplayer-commercial build SUCCESS
[:clipboard:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/24013/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-commercial-unit-tests/24013/consoleFull) jwplayer-commercial unit tests SUCCESS
[:cucumber: Cucumber Integration Tests](https://jenkins-player.longtailvideo.com/job/jw7-commercial-cucumber-wdio/6194/allure/) UNSTABLE
[:jack_o_lantern:](http://player-pr-test-jenkins.longtailvideo.com/builds/28583/archive/test/squash/) Manual Tests
[:tv:](http://player-pr-test-jenkins.longtailvideo.com/builds/28583/archive/test/view/) Views

File: src/js/controller/model.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ export type PlayerModelAttributes = {
     nextUp: PlaylistItem;
     nextUpDisplay: boolean;
     pauseReason: PauseReason;
+    pip: boolean;
```
Comment (zetagame) [inline src/js/controller/model.ts:74]: this should already be on the model. Issue with base branch?

**Review by zetagame — COMMENTED** [review]
Hmm, it looks like all the changes in this PR are already in master. Should fetch and reset to re-assess what's causing the issue.

File: src/js/controller/model.ts (hunk 1 above)
Comment (maniland) [reply src/js/controller/model.ts]: merged master fixes this issue but my change now appears on master I think I mat have made a mistake here looking into it 👍

Comment (maniland) [reply src/js/controller/model.ts]: looks like I didn't have an up to date master and Jong had already fixed this bug with the same change I'll delete the PR 👍

**Pull request closed without merging**